certbot/tests/main_test.py

```diff
@@ -1,48 +1,68 @@
 """Tests for certbot.main."""
+import os
+import tempfile
 import unittest
 
-
 import mock
-
+import six
 
 from certbot import cli
 from certbot import configuration
+from certbot import errors
+from certbot import main
 from certbot.plugins import disco as plugins_disco
 
 
 class ObtainCertTest(unittest.TestCase):
```

Comment:
> 18:24 < TheNavigat> The fact that someone decided that the default method in 
                    main_test's _call should be obtain_cert is sad

This is because these tests are for `ObtainCert`.

You should make a new `MainTest` class instead of modifying this one.